doc/backend/branding.md

```diff
@@ -136,3 +136,89 @@ like `@page`. Supported css features depending on weasyprint, see [here](https:/
+  - If icon_text and icon_image are set, only image will be displayed.
+  - Example: `"icon_image": "https://www.tracim.fr/static/images/new_tracim/LOGO_TRACIM_RVB_1.png"`
+- `content_type_filter`:
+  - The content type on which the App Custom Action will be available.
+  - Must be a list of comma separated content type
+  - Available values are: 'html-document', 'file', 'thread', 'kanban', 'logbook'.
+  - Example: `"content_type_filter": "file,thread,kanban"`
+- `content_extension_filter`:
+  - The content extension on which the App Custom Action will be available.
+  - Must be a list of comma separated extension containing the dot '.'
+  - Example: `"content_extension_filter": '.jpg,.png,.gif'`
+- `content_label_filter`:
+  - The content label pattern on which the App Custom Action will be available.
+  - Must be string that will be matched as a regex. Case-insensitive.
+  - Example: `"content_label_filter": "some"`
+    - will match "some label", "another some label" but not "last label"
+- `workspace_filter`:
+  - The workspace id on which the App Custom Action will be available.
+  - Must be a list of comma separated ids
+  - Example: `"workspace_filter": "2,10,42"`
+- `user_role_filter`:
+  - The user's roles on whom the App Custom Action will be available.
+  - Must be a list of comma separated roles
+  - Available values are: workspace-manager, content-manager, contributor, reader
+  - Example: `"user_role_filter": "workspace-manager,content-manager,contributor"`
+- `user_profile_filter`:
+  - The user's profiles on whom the App Custom Action will be available.
+  - Must be a list of comma separated profiles
+  - Available values are: administrators, trusted-users, users
+  - Example: `"user_profile_filter": "administrators,trusted-users"`
+- `link`:
+  - The on click https link the submenu option will redirect to.
+  - You can use variables to transfert data to the link destination. Place each of them inside braces.
+  - Available variables are:
+    - "content.label": The label of the current content. Url encoded.
+    - "content.content_id": The id of the current content.
+    - "content.workspace_id": The workspace id of the current content.
+    - "content.author_id": The author id of the current content.
+    - "content.author_name": The author name of the current content. Url encoded.
+    - "content.url": The url to open the content in tracim. Url encoded.
+    - "user.user_id": The id of the currently connected user. The one that will click on the App Custom Action.
+  - Example: `"link": "https://some.domaine.com/open?content_label={content.label}&content={content.content_id}&space_id={content.workspace_id}`
+- `label`:
+  - An object containing the translation keys of the submenu option. The keys are the language id and the
+  values are the translated string.
+  - Available key values are: en, fr, ar, de, eo, es, it, pt
+  - Example: `"label": { "en": "my english label", "fr": "mon label français" }`
```

Comment:
Give example with same structure like visible in first example (multiple line ).

Reply:
will fix

Reply:
fixed